WordPress/src/main/java/org/wordpress/android/ui/posts/mediauploadcompletionprocessors/MediaUploadCompletionProcessorPatterns.java

```diff
@@ -11,7 +11,7 @@ public class MediaUploadCompletionProcessorPatterns {
     public static final Pattern PATTERN_BLOCK_HEADER = Pattern.compile(new StringBuilder()
             .append(PATTERN_BLOCK_PREFIX)
             .append(MediaBlockType.getMatchingGroup())
-            .append(").*? -->\n?")
+            .append(").*? (/?-->)\n?")
```

Comment:
This new capture group is used to determine if the block uses the self-closing tag.